src/network/core/game_info.h

```diff
@@ -50,6 +89,19 @@ struct NetworkGameInfo : NetworkServerGameInfo {
 	byte map_set;                                   ///< Graphical set
 };
 
-const char * GetNetworkRevisionString();
+extern NetworkServerGameInfo _network_game_info;
+
+const char *GetNetworkRevisionString();
+bool IsNetworkCompatibleVersion(const char *other);
+
+void FillNetworkGameInfo(NetworkGameInfo &ngi);
+
+void ReceiveGRFIdentifier(Packet *p, GRFIdentifier *grf);
+void SendGRFIdentifier(Packet *p, const GRFIdentifier *grf);
+
+void ReceiveNetworkGameInfo(Packet *p, NetworkGameInfo *info);
+void SendNetworkGameInfo(Packet *p, const NetworkGameInfo *info);
+
+void HandleIncomingNetworkGameInfoGRFConfig(GRFConfig *config);
```

Comment:
Is only used in game_info.cpp, so why make it a "public" function, just make it a static function in game_info.cpp.